lib/iris/tests/runner/_runner.py

```diff
@@ -181,6 +181,10 @@ def run(self):
         args = ['', None, '--processes=%s' % n_processors,
                 '--verbosity=2', regexp_pat,
                 '--process-timeout=250']
+        try:
+            import gribapi
+        except ImportError:
+            args.append('--exclude=^grib$')
```

Comment:
NB. This matches the directory name "grib" when nose is doing test discovery and so excludes all of the tests under `lib/iris/tests/unit/fileformats/grib/...`. The alternative was to modify all those tests to add `skip_grib`.... :unamused: